tests/stdlib/tos.nim

```diff
@@ -138,4 +139,53 @@ let tm = fromUnix(0) + 100.microseconds
 writeFile("a", "")
 setLastModificationTime("a", tm)
 echo getLastModificationTime("a") == tm
-removeFile("a")
\ No newline at end of file
+removeFile("a")
+
+block normalizedPath:
+  doAssert normalizedPath("/a/b/../../../foo", '/') == "/foo"
+  doAssert normalizedPath("\\a\\b\\..\\..\\..\\foo", '\\') == "\\foo"
+
+when defined(Linux) or defined(osx):
```

Comment:
That's cheating. You can't add something to ``os`` and only test it on Linux/Mac.